src/ebonite/build/builder/docker_builder.py

```diff
@@ -12,7 +12,9 @@
 from jinja2 import Environment, FileSystemLoader
 
 from ebonite.build.builder.base import PythonBuilder, ebonite_from_pip
+from ebonite.build.docker_objects import DockerImage
```

Comment:
maybe we need to change package/module layout here? smth like ebonite.build.core.runner.Runner and ebonite.build.docker.runner.DockerRunner or something? Just an idea

Reply:
I don't see any difference here. I expect that Ebonite user wouldn't have to be in touch with this code directly. Thus why should we care about minor naming things?

Reply:
It's not about user experience, but about reasonable logic and structure in module paths and names. Same goes about word simple - it's just redundant, there is no complex_docker. The point of renaming and restructuring is to make it easy to find everything

Reply:
I eliminated word "simple". More complex changes are postponed for `EBNT-245-instance` branch works (I refactor this part of code there so it seems unreasonable to groom layout until code stabilizes).